sys/include/net/ieee802154/radio.h

```diff
@@ -382,6 +383,14 @@ typedef struct {
  * @brief Radio ops struct declaration
  */
 struct ieee802154_radio_ops {
+    /**
+     * @brief Radio device capabilities
+     *
+     * This field contains bitflags of supported capabilities
+     * (@ref ieee802154_rf_caps_t) by the device.
+     */
+    uint16_t caps;
```

Comment:
since this won't change, it can be declared as `const`
```suggestion
    const uint16_t caps;
```